ts/components/CodeInput.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  CodeInput as IOCodeInput,
+  WithTestID
+} from "@pagopa/io-app-design-system";
+import { useFocusEffect } from "@react-navigation/native";
+import React from "react";
+import { StyleSheet, View, TextInput } from "react-native";
+import { Millisecond } from "@pagopa/ts-commons/lib/units";
+import { setAccessibilityFocus } from "../utils/accessibility";
+
+type CodeInputProps = WithTestID<Parameters<typeof IOCodeInput>[0]> & {
+  onChange: (text: string) => void;
+  accessibilityLabel: string;
+  onFocusedA11yView?: React.RefObject<View>;
+};
+
+const FOCUS_TIMEOUT = 1000;
+const A11Y_FOCUS_TIMEOUT = FOCUS_TIMEOUT + 200;
+const FOCUS_TIMEOUT_MS = FOCUS_TIMEOUT as Millisecond;
+const A11Y_FOCUS_TIMEOUT_MS = A11Y_FOCUS_TIMEOUT as Millisecond;
+
+/**
+ * Temporary wrapper to DS CodeInput to allow keyboard opening on item focus.
+ */
+export const CodeInput = ({
+  onChange,
+  accessibilityLabel,
+  testID,
+  onFocusedA11yView,
+  ...props
+}: CodeInputProps) => {
+  const hiddenInputRef = React.useRef<TextInput>(null);
+
+  useFocusEffect(
+    React.useCallback(() => {
+      const timeoutId = setTimeout(() => {
+        hiddenInputRef.current?.focus();
+      }, FOCUS_TIMEOUT_MS);
+
+      if (onFocusedA11yView) {
+        setAccessibilityFocus(onFocusedA11yView, A11Y_FOCUS_TIMEOUT_MS);
+      }
+
+      return () => {
+        clearTimeout(timeoutId);
+      };
+    }, [onFocusedA11yView])
+  );
+
+  return (
+    <View
+      accessible
+      accessibilityLabel={accessibilityLabel}
+      accessibilityElementsHidden
+    >
+      <TextInput
+        testID={testID}
```

Comment:
```suggestion
        autofocus={true}
        testID={testID}
```
Considering the [figma](https://www.figma.com/design/BEXvjQmBkMRRFiNcMgE8g3/Login-Veloce?node-id=4387-76442&t=OCaLCTMZbSeiLxMG-4) it seems to me that we need to add autofocus, we can try asking the design team

Reply:
Thank you for your suggestion! I tried this solution, but since this component is displayed after `IdentificationModal` it seems like the autofocus isn't behaving correctly. This scenario has forced me to manually trigger the input focus after a delay.